macros/field_extractions/get_page_view_context_fields.sql

```diff
@@ -33,7 +33,7 @@ You may obtain a copy of the Snowplow Personal and Academic License Version 1.0
             {{ snowplow_utils.get_optional_fields(
             enabled=true,
             fields=page_view_fields(),
-            col_prefix='contexts_com_snowplowanalytics_snowplow_web_page_1_',
+            col_prefix='contexts_com_snowplowanalytics_snowplow_web_page_1',
```

Comment:
It seems like there are other field extractions which need fixing, can you please add them as well? 
<img width="715" alt="image" src="https://github.com/user-attachments/assets/ff0b8d75-4453-4c6e-b90a-b397035bfe18" />